app/views/pages/_privacy_text.html.erb

```diff
@@ -1,4 +1,4 @@
-<p>Effective May 25, 2018</p>
+<p>Effective Dec 20, 2018</p>
```

Comment:
Going out on a limb... if we change the privacy policy today, we should include today's date as the last update (to the policy, not to our behavior). As a user, having read the terms last week, if it says last update 2018, I'll assume there has been no change.

Reply:
Yeah that's probably more reasonable than changing it the "last effective date" that I was thinking. cc @pkfrank since he might have thoughts on this.